common/util.go

```diff
@@ -431,12 +432,22 @@ func CreateHistoryStartWorkflowRequest(
 	domainID string,
 	startRequest *types.StartWorkflowExecutionRequest,
 	now time.Time,
-) *types.HistoryStartWorkflowExecutionRequest {
+) (*types.HistoryStartWorkflowExecutionRequest, error) {
 	histRequest := &types.HistoryStartWorkflowExecutionRequest{
 		DomainUUID:   domainID,
 		StartRequest: startRequest,
 	}
-	firstDecisionTaskBackoffSeconds := backoff.GetBackoffForNextScheduleInSeconds(startRequest.GetCronSchedule(), now, now)
+
+	firstDecisionTaskBackoffSeconds := backoff.GetBackoffForNextScheduleInSeconds(
```

Comment:
I am thinking:
startTime := now.Add(time.Second * delayStartSeconds)
firstDecisionTaskBackoffSeconds := backoff.GetBackoffForNextScheduleInSeconds(startRequest.GetCronSchedule(), startTime, startTime)

So. you don't need to return an error.

Reply:
I think there's more than to this than how to implement it. We need to clearly explain this behavior. Let's talk about this behavior in an ERD after this change then update here.